pkg/shp/resource/resource.go

```diff
@@ -0,0 +1,80 @@
+package resource
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+
+	"github.com/shipwright-io/cli/pkg/shp/params"
+	"github.com/shipwright-io/cli/pkg/shp/util"
+)
+
+type ShpResource struct {
```

Comment:
I see a number of "shp" being used everywhere, and we don't need to repeat the project name in every underlying type. So, could we use direct naming like `Resource` in this case?

Reply:
fixed